nielslange/woo-test-environment
# Correct shop URL for empty cart

Fixes #21 

Currently, when the cart is empty, there's a link saying `Browse store`, which links to `http://localhost:8889/shop/`. This link works as long as someone spins up the site using [@wordpress/env](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-env/) with the default settings. However, this link will no longer work when spinning up the site in a non `@wordpress/env` environment. This PR aims to change the URL of the `Browse store` link from `http://localhost:8889/shop/` to `/shop/`. This way, the `Browse store` link will work independent of the site URL and the site protocol (`http://` or `https://`).

## Testing steps

- Check out this PR on a non `@wordpress/env` environment, e.g. Local WP or Laravel Valet.
- Go to the cart page, without having any products added to the cart.
- Click on the `Browse store` link.
- Verify that the link leads to the shop page, which is not located on `http://localhost:8889/shop/`.
